src/ar/ARTester.cpp

```diff
@@ -0,0 +1,94 @@
+#include "Detector.h"
+
+#include <opencv2/core.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+
+#include <iostream>
+
+using namespace std;
```

Comment:
This is bad practice. Also even though this is used, there is still a mix of "cout" and "std::cout" below.